source/shared/cpp/ObjectModel/TextBlockText.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <string>
+#include <vector>
+#include "Enums.h"
+#include "TextSection.h"
+#include "TextSection.h"
+
+namespace AdaptiveCards
+{   
+    class TextBlockText
```

Comment:
name is confusing.
DateTimeFormatter may be?

Reply:
I agree that the name should be changed for TextBlockText and TextSection but I think DateTimeFormatter is not a good name as this class doesn't format the date and time, it just splits the text and finds possible matches for a date or time

Reply:
We have the names DateTimePreparser and DateTimePreparserToken right now, if someone has a better name I can change them